src/pagination/index.tsx

```diff
@@ -12,17 +20,27 @@ import { ListOption } from '../list';
 import bundle from './Pagination.nls';
 import * as css from '../theme/default/pagination.m.css';
 
-function createMemoryTemplate<S = void>(): DataTemplate<S> {
+export function createMemoryTemplate<S = void>({
+	filter
+}: { filter?: (query: ResourceQuery[], v: S) => boolean } = {}): DataTemplate<S> {
 	return {
-		read: (_, put, get) => {
+		read: ({ query }, put, get) => {
```

Comment:
what made you add this in? I don't believe you're using the filter or query at all are you?

Reply:
No, I just copied it from examples.